sql/hive/src/main/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -51,6 +51,11 @@ object TestHive
         // SPARK-8910
         .set("spark.ui.enabled", "false")))
 
+trait TestHiveSingleton {
+  protected def sqlContext: SQLContext = TestHive
+  protected def hiveContext: HiveContext = TestHive
```

Comment:
alternatively if you make this extend `SQLTestUtils` then subclasses can just `import testImplicits._`, which is more consistent with other test suites.